docs/arcaflow/workflows/flow-control.md

```diff
@@ -2,6 +2,258 @@
+
+Here is a visual demonstrating two steps that have opposite logic:
+```mermaid
+stateDiagram-v2
+  classDef tag font-style:italic;
+  classDef step font-weight:bold;
+
+  state if_state <<choice>>
+  Step1:::step: Step 1
+  [*] --> Step1
+  Step1 --> if_state
+  Step2:::step: Step 2
+  Step3:::step: Step 3
+  if_state --> Step2: !expr $.step1.output_1 == true
+  if_state --> Step3: !expr $.step1.output_1 == false
+  oneof:::tag: !oneof
+  Step2 --> oneof
+  Step3 --> oneof
+  oneof --> [*]
+```
+
+Here is a visual demonstrating utilizing the disabled output:
+```mermaid
+stateDiagram-v2
+  classDef tag font-style:italic;
+  classDef step font-weight:bold;
+
+  input
+  [*] --> Step1:::step
+  input --> Step1: !expr $.input.enabled
+  or_disabled:::tag: !ordisabled
+  Step1 --> or_disabled: success
+  Step1 --> or_disabled: Disabled
+  or_disabled --> [*]
+```
+
+The way this works is that either of the two paths can create a valid output, allowing the workflow to continue past the disabled steps.
+
+##### How to do this in a workflow
+
+There are two tags that create the described OR dependency.
+
+| Tag           | Description                                      |
+|---------------|--------------------------------------------------|
+| `!oneof`      | A general use tag to depend on any of the inputs |
+| `!ordisabled` | A special case of `!oneof` that automatically handles the disabled case |
+
+##### How to use `!oneof`
+
+To use `!oneof` for graceful handling of disabled steps, the oneof should depend on either ONE OF the success output or the disabled output.
+
+The oneof tag is a method of creating a schema `one_of_string` type from values present in the workflow.
+
+The syntax of `!oneof` is:
+- Following the tag `!oneof`, create a new YAML section (map) by moving to a new line. That section should contain two properties:
+  - discriminator: A string that specifies what the oneof discriminator should be.
+  - options: A YAML section (map) that contains all options. The keys are the discriminator values, and the values should be valid expressions.
+
+Example:
+```yaml title="workflow.yaml"
+version: v0.2.0
+input:
+  root: RootObject
+  objects:
+    RootObject:
+      id: RootObject
+      properties:
+        step_1_enabled:
+          type:
+            type_id: bool
+        step_1_input:
+          type:
+            type_id: integer
+steps:
+  my_step:
+    plugin:
+      deployment_type: image
+      src: path/to/my_plugin:1
+    input:
+      param_1: !expr $.input.step_1_input
+    enabled: !expr $.input.step_1_enabled
+outputs:
+  workflow_success:
+    plugin_output: !oneof
+      discriminator: result
+      options:
+        enabled: !expr $.steps.my_step.outputs.success
+        disabled: !expr $.steps.my_step.disabled.output
+```
+
+##### How to use `!ordisabled`
+
+Since arcaflow needed a simple way to disable a step easily, there is a method that is as simple as changing the expression tag from `!expr` to `!ordisabled`
```

Comment:
"arcaflow needed" suggests an agency that doesn't exist, and "simple way [...] to [...] easily" is redundant. How about "Arcaflow provides workflow developers an easy way ..."?

Reply:
Done. I also added more information regarding the !ordisabled tag.